phpunit.xml.dist

```diff
@@ -9,6 +9,7 @@
     verbose="true"
     colors="auto"
     bootstrap="vendor/autoload.php"
+    failOnWarning="true"
```

Comment:
Not based on my tests (as seen above). When the setting isn't set, PHPUnit does complain about the warning, but the exit code is `0` (success), which is what we saw happen in #1318 (the travis build succeeded even though the warning was encountered).  With that config option set (or the `--fail-on-warning` CLI flag is set) the PHPUnit complains **and** sets the exit code to `1`.

Here's what I can find in the PHPUnit source:

https://github.com/sebastianbergmann/phpunit/blob/e083cb6682159fcd7863464729211aae22c1818a/src/Util/Configuration.php#L700

https://github.com/sebastianbergmann/phpunit/blob/e083cb6682159fcd7863464729211aae22c1818a/src/TextUI/Command.php#L109

https://github.com/sebastianbergmann/phpunit/blob/e083cb6682159fcd7863464729211aae22c1818a/src/TextUI/Command.php#L510